src/applications/vaos/components/TabNav.jsx

```diff
@@ -9,14 +11,23 @@ class TabNav extends React.Component {
       <ul className="va-tabs vaos-appts__tabs" role="tablist">
         <TabItem
           id="upcoming"
-          shortcut={1}
           tabpath="/"
+          isActive={this.props.location.pathname === '/'}
+          firstTab
+          onNextTab={() => {
```

Comment:
Normally I think we'd come up with a more scalable approach for this, but we're only going to have two tabs and this is pretty simple.